contracts/Rules.sol

```diff
@@ -1,23 +1,14 @@
 pragma solidity >=0.4.22 <0.6.0;
 
-import "./AdminProxy.sol";
 import "./RulesProxy.sol";
 import "./Ingress.sol";
+import "./Admin.sol";
 
 
-contract Rules is AdminProxy, RulesProxy {
-    // These will be assigned at the construction
-    // phase, where `msg.sender` is the account
-    // creating this contract.
-    address public admin = msg.sender;
+contract Rules is RulesProxy, Admin {
```

Comment:
Should this still be an admin?

Reply:
I made it an Admin so we can use the modifiers.

The other approach would be making the Rules contract aware of the Admin contract (with an address in the constructor).

Thoughts?

Reply:
It should be looking up the admin contract through the ingress contract and then calling the admin contract.

Reply:
you won't be able to use the modifiers then though?

Reply:
I'll create a modifier that delegates to the Admin contract.

Reply:
👍